docker-compose.yml

```diff
@@ -68,7 +68,7 @@ services:
           - node.labels.${STACK_NAME?Variable not set}.app-db-data == true
 
   backend:
-    image: 'ghcr.io/project-chip/csa-certification-tool-backend:0a12467'
+    image: 'ghcr.io/project-chip/csa-certification-tool-backend:52827eb'
```

Comment:
Will there be another image once the backend PR is merged and then this will be updated? Or will the backend SHA for the image be from the feature branch instead of the merge commit from that PR?